src/Forms/FixerFormType.php

```diff
@@ -0,0 +1,41 @@
+<?php
+
+/*
+ * This file is part of the JoliTypo Website project.
+ * (c) JoliCode <coucou@jolicode.com>
+ * For the full copyright and license information, please view the LICENSE
+ * file that was distributed with this source code.
+ */
+
+namespace App\Forms;
```

Comment:
This is not the standard for Type
It should be `App\Form` or better `App\Form\Type`